### ISSUE TITLE
CouchDB connection request does ignore subfolders in Server URL field

### ISSUE THREAD

**Issue by thrdroom** [open-nomie/nomie6-oss#19]
**The Bug:**
Im running couchdb behind an nginx reverse proxy which is exposed via a subfolder:

**example:** https://mydomain.com/mycouchdb/

Database name is **mynicedb**
Username is **myusername**

when i try to connect nomie to the couchdb i enter **https://mydomain.com/mycouchdb/** as my server url, and **mynicedb** as my database name.

Nomie then gives an error `NetworkError when attempting to fetch resource` because nomie does try to connect via the following url **https://mydomain.com/mynicedb**. It does completely ignore the subfolder /mycouchdb/ from the server url field.

**Expected behaviour:**
Nomie should honor subfolders in the server url field.

**Comment by donalmacanri** [comment]
Can you try entering https://mydomain.com as your server url and *mycouchdb/mynicedb* as your database name?

**Comment by thrdroom** [comment]
> Can you try entering https://mydomain.com as your server url and **mycouchdb/mynicedb** as your database name?

yes, this way was actually my workaround to get the app running. The issue should still be addressed for new users, who don't know how to debug the error message.

**Comment by donalmacanri** [comment]
Perhaps it would be less confusing to have a single text field for the full CouchDb URL (instead of having a separate field for database name)?

**Comment by thrdroom** [comment]
> Perhaps it would be less confusing to have a single text field for the full CouchDb URL (instead of having a separate field for database name)?

No, i think that would be even more confusing if users would have to figure out how to structure the URL correctly.
The "database name" field itself is fine, and not really confusing. But the field should just be used for the databasename and not parts of the "Server URL". The problem is that the "Server URL" field does strip away everything from an URL that comes after the first **/** behind the domains tld-ending.

The filter of the "Server URL" has to be modified to respect subdirectories that are put into the field and not only the domain name and domainending.
